go.mod

```diff
@@ -57,10 +57,11 @@ require (
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
+	github.com/varlink/go v0.4.0
```

Comment:
You've re-added Varlink somehow - I think this needs a fresh revendoring

Reply:
Yes most likley you have some file in your go source that is no longer in master.  
grep -r github.com/varlink .

Should find the file causing this.

Reply:
Tried that last night and didn't spot anything.  I badly need to upgrade my sandbox VM, I think this is a good excuse to do so and try vendoring again in a fresh environment.  I'm going to close this one.